console/src/helpers/Html.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import { env } from './localDev';
+/**
+ * Wrapper component containing HTML metadata and boilerplate tags.
+ * Used in server-side code only to wrap the string output of the
+ * rendered route component.
+ *
+ * The only thing this component doesn't (and can't) include is the
+ * HTML doctype declaration, which is added to the rendered output
+ * by the server.js file.
+ */
+
+type IAssets = {
```

Comment:
```suggestion
type Assets = {
```

As we stick to not prefixing type aliases or interfaces with `I`.